pyproject.toml

```diff
@@ -26,3 +26,4 @@ aiohttp-json-rpc = "0.11.2"
 ujson = "1.35"
 toml = "0.9.4"
 pysha3 = "1.0.2"
+pytest = "4.0.1"
```

Comment:
Please move that to the  `tool.poetry.dev-dependencies` section.

Reply:
@eugene-babichenko, done.